ignite/engine/events.py

```diff
@@ -179,6 +182,33 @@ def call_once(engine):
     GET_BATCH_STARTED = "get_batch_started"
     GET_BATCH_COMPLETED = "get_batch_completed"
 
+    def __or__(self, other):
+        return EventList() | self | other
+
+
+class EventList:
```

Comment:
Shouldn't we inherit of something instead of introducing new type of events ?

Reply:
I didn't introduce a new type of event but a list of event with operator `__or__`.